users/tests.py

```diff
@@ -93,5 +104,110 @@ def test_signupview_post_invalid_keys(self):
             }
         )
 
+class HostTest(TestCase):
+    def setUp(self):
+        User.objects.create(
+            id          = 1,
+            kakao_id    = 1,
+            name        = 'anhesu',
+            profile_url = 'testurl',
+            email       ='anhesu1@naver.com',
+            password    = '1234'
+        )
+        Host.objects.create (
+            id          = 1,
+            nickname    = 'anhesu',
+            profile_url = 'testurl',
+            user_id     = 1
+        )
+        self.client.head
     def tearDown(self):
-        User.objects.all().delete()
\ No newline at end of file
+        Host.objects.all().delete() 
+
+    def test_hostview_get_success(self):
+        client = Client(HTTP_USER_AGENT="Mozilla/5.0 ...", HTTP_Authorization = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoxfQ.Rycaz6XldIy-q-xnORvTDQcdfOvEuTrE1BSSf74AiEQ")
```

Comment:
token을 미리 생성한 후에 하드코딩해서 사용하는 방법인가요?
이렇게 되면, 서버에사 사용하고 있는 `SECRET_KEY` 또는 `ALGORITHM`이 변경될 경우 테스트코드도 영향을 받게 될 것 같습니다.

직접 로그인 엔드포인트를 호출해서 토큰을 받아온 이후에, host 등록 엔드포인트를 호출하는 방향으로 수정 부탁드립니다!
 

```suggestion
        client = Client(HTTP_USER_AGENT="Mozilla/5.0 ...", HTTP_Authorization = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoxfQ.Rycaz6XldIy-q-xnORvTDQcdfOvEuTrE1BSSf74AiEQ")
```